text/0000-interactive-app-creation.md

```diff
@@ -0,0 +1,142 @@
+- we have not added a "skip" option here, because the purpose is to focus on improved accessibility in Ember apps.
+
+#### Package Manager
+
+- ignore/skip has been added to cover the use case where the developer is in a workspace 
+
+#### CI Option
+
+Separate RFCs should further define more options for CI. 
+  
+### Current Limitations
+Explicitly, the goal of this RFC is to only include `ember-cli` flags that already exist. Any other possibilities should be considered future work.
+
+This interactive workflow will also ignore any ~/.ember-cli settings that already exist. A separate RFC should be written that makes it possible for these to things to co-exist and respect the settings of the other.
+
+### Future possibilities
+An example of a future RFC that could be specific to addons, would be to add a `--supported-version` flag for addons and add that to the wizard: 
+
+- `What is the earliest version of Ember that you intend to support? (use arrow keys)`
+  - `> recommended (last two LTS)`
+  - `last LTS`
+  - `manually define a version number`
+
+### Implementation
+This section is not normative, but provided for extra context for folks who might be new to this idea in general. We could use something like [inquirer.js](https://github.com/SBoudrias/Inquirer.js) to implement this wizard, if the mechanism doesn't already exist within the ember-cli codebase. However, it should be explicitly noted that inquirer is only being used as an example of a library that could be used, and this RFC isn't explicitly defining that inquirer.js should be or will be used. 
+
+## How we teach this
+
+As this is a wholly new idea, it should be documented and added to the guides along with screenshots of the new workflow. Until that prototype exists, this section will largely remain empty. However, we intend to explain it similarly to Vue's documentation for a similar feature (https://cli.vuejs.org/guide/creating-a-project.html#vue-create).  
+
+## Drawbacks
+
+It could be considered a drawback to make this not the default for new Ember apps, even if an app name is defined. 
+
+The general drawback with wizards is that it can lead to a large number of combinations, which increases the risk for fragmentation. We believe that we have limited this risk by limiting the number of questions asked, but it is still a risk and should be identified as such.
+
+## Alternatives
+
+1. We could decide on a different list of questions. See the notes from the [strike team discussion on May 20th](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-05/may-20.md) to review the other possibilities considered.
+1. We could make this the default for new Ember apps and add it in the appropriate version as a change.
+1. We could have this workflow be opt-in only, via the `--interactive` flag (e.g., `ember new --interactive`). Using the `ember new` command would error and trigger the help workflow (which would be updated to include the `--interactive` flag).
```

Comment:
I'm always up for "sane defaults", which from my experience makes the flow more straightforward, expected (guessable) and more pleasant to use. In this case I see the sane default option the originally proposed one `ember new` => wizard.